ansible/roles/discover/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Install discovery-related packages
+  apt: name={{ item }} state=latest
+  with_items:
+    - avahi-daemon
+    - avahi-utils
+    - libnss-mdns
+    - dnsmasq
+
+- name: Configure avahi-daemon
+  template: src=avahi/avahi-daemon.conf
```

Comment:
Did we decide on preferring yaml or string format? I think we settled on YAML. (#11)